pkg/apis/release/v1alpha1/types.go

```diff
@@ -204,6 +204,27 @@ const (
 	ConditionPayloadRejected string = "PayloadRejected"
 )
 
+// These are valid Reason values for the ReleasePayloadStatus Conditions
```

Comment:
These aren't API, move them to your controller.

Reply:
Done!